src/sentry/templates/sentry/vercel-ui-hook.vercel

```diff
@@ -0,0 +1,9 @@
+<Page>
+{% block main %}
+    <Box display="flex" justifyContent="space-between">
+      <P>
+        • <Link href="https://www.sentry.io/settings/{{ org }}/integrations/vercel/{{ integration_id }}/">Add to project for {{ org }}</Link><BR />
```

Comment:
@ceorourke Oh, I just realized you have Sentry's URL hardcoded here. We should never do that. Instead, use `absolute_uri` which will work for on-prem users as well.